api/lib/application/users/index.js

```diff
@@ -64,7 +64,7 @@ exports.register = async function(server) {
             return h.response(jsonApiError).code(errorHttpStatusCode).takeover();
           }
         },
-        handler: userController.getUserDetailForAdmin,
+        handler: userController.getUserDetailsForAdmin,
```

Comment:
Personnellement, je ne sais pas si on doit parler du détail d'un utilisateur ou des détails.
[Et Internet ne répond pas clairement.](https://english.stackexchange.com/questions/3675/is-details-singular/3681#3681). On se dit qu'on n'utilisera plus jamais le singulier sur détail dans le futur ?

Reply:
La question que tu as liée se pose la question de l'accord du verbe `here is the details/here are the details` mais pas de l'orthographe du mot qui est toujours `details`. `detail` en anglais peut désigner _un détail_ mais pas _le détail_ d'un objet. Donc en anglais pas d'ambiguité, c'est bien `getUserDetails` qui est correct. Sauf si l'objet avait une collection de détails et qu'on en récupérait un unique avec `getUserDetail(userId, detailId)`…

Reply:
Merci pour les précisions, on en a discuté lors du point tech aussi sur l'orthographe detail/details :) et on a conclu que plutôt c'est details / @octo-pito t'étais en team captain